src/wallet3/rpc/request_handler.cpp

```diff
@@ -50,9 +54,39 @@ using cryptonote::rpc::rpc_error;
 const std::unordered_map<std::string, std::shared_ptr<const rpc_command>> rpc_commands = register_rpc_commands(wallet_rpc_types{});
 
 void RequestHandler::invoke(GET_BALANCE& command, rpc_context context) {
+  if (auto w = wallet.lock())
+  {
+    command.response["balance"] = w->get_balance();
+    command.response["unlocked_balance"] = w->get_unlocked_balance();
+  }
```

Comment:
should handle else case here

Reply:
Had a chat to Tom about this, once we destroy the wallet object there may be pending RPC requests on the omq job queue. Intention is If the wallet is gone, just ignore them.